wikidp/utils/__init__.py

```diff
@@ -220,38 +249,43 @@ def get_lang(_dict, default=None):
 
     Args:
         _dict (dict): Dictionary for getting value
-        default (optional): Expected return if value does not exist for fallback language
+        default (optional): Expected return if value does not exist for
+            fallback language
 
     Returns: value of dictionary's language key or default
     """
     if not _dict:
-        pass
-    value = _dict.get(LANG)
-    if value:
-        return value
-    return _dict.get(FALLBACK_LANG, default)
+        return default
+    return _dict.get(WIKIDATA_LANG) or _dict.get(WIKIDATA_FB_LANG, default)
 
 
 # pylint: disable=R0914
 def item_detail_parse(qid, with_claims=True):
     """
-    Use the JSON representation of wikidataintegrator to parse the item ID specified.
+    Get Wikidata information by QID.
 
-    Returns a new dictionary of previewing information and a dictionary of property counts.
+    Returns (Dict): overview of key information
     """
     item = wd_int_utils.get_item_json(qid)
     if not item:
         return False
-    label = parse_wd_response_by_key(item, 'labels', default="Item {}".format(qid))
+    label = parse_wd_response_by_key(item, 'labels', default=f"Item {qid}")
     aliases = parse_wd_response_by_key(item, 'aliases', default=[])
     description = parse_wd_response_by_key(item, 'descriptions', default='')
-    output_dict = {'qid': qid, 'label': label, 'aliases': aliases, 'description': description}
+    output_dict = {
+        'aliases': aliases,
```

Comment:
The key names here might be good candidates for constants also

Reply:
Agreed!
I'm planning to refactor quite a bit of this file to make the code a bit easier to follow as well (more object-oriented). I'll be sure to keep this comment in mind when I do!